manifests/stage-ofed-driver/0300_rolebinding.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: ofed-driver
+  namespace: {{ .RuntimeSpec.Namespace }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: ofed-driver
+  namespace: {{ .RuntimeSpec.Namespace }}
+subjects:
+- kind: ServiceAccount
+  name: ofed-driver
+  namespace: {{ .RuntimeSpec.Namespace }}
+userNames:
```

Comment:
is the `userNames` attribute intended for older versions of K8s ?  i see the V1 api does not have this attribute[1]

[1] https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#rolebinding-v1-rbac-authorization-k8s-io